### DISCUSSION TITLE
Are circuit group controlled relays robustly synced?

### DISCUSSION THREAD

**Discussion by ddyer8** [tagyoureit/nodejs-poolController#490] [Q&A] [answered]
I have 120VAC overhead lighting around my pool, which I'd like to control with Nixie.  To ensure current margin through the sequent 8-relay board, I'm splitting the load across 2-relays.  In REM, I have them set as "Overhead Lights 1" and "Overhead Lights 2."  In Nixie, I have the 2 relays setup as Generic Aux Circuits.  Then I created an "Overhead Lights" circuit group, which controls the 2 circuits to turn on the lights.  

So the question is... Will the system consistently set the relays on/off simultaneously or is there a better way to execute this?

**Comment by rstrouse** [comment]
Are these incandescent lights?  If they are LED 8A is a whole hell of a lot of light.  Maybe in a French fry warmer or mercury vapor lighting you could get up there but if you are over 8A two relays will not alleviate your load.  Put another way you cannot make a baby in 4.5 months by adding another mother.

**Comment by ddyer8** [comment]
Yes, there are several strands of incandescent bulbs and my current setup I think only draws maybe 3A, but There is another outlet on the circuit so I’m trying to accommodate up the the circuit protection. Also The sequent folks only rate the traces on the PCB up to 4A (contacts are 8A and relay is 10A). 

So yes, I can run the current setup on a single relay but if I can reliably sync the relays, I’d like to use another channel to have a more robust setup. 

Thank you,
Dustin

**Comment by rstrouse** [comment]
The switching on the relays can vary by a few hundred milliseconds.  Even so when not mechanically switched you will find dual relays to cause contact burn.  You could however solder additional wire between the pins on the relay and the connector.

**Reply by ddyer8** [reply]
Copy.  I'll pursue an external relay or bypass the traces.

**Comment by rock-crusher** [comment, marked as answer]
I would suggest just adding an auxiliary relay as suggested in the [wiki](https://github.com/tagyoureit/nodejs-poolController/wiki/DIY-Standalone-Nixie-Pool-Controller#high-current-circuits), by doing so, your covered for most loads, and should a short circuit happen in your light circuit it won’t blow-out your Sequent relay board. 

![Light Relay](https://user-images.githubusercontent.com/94593954/174501095-17ccf1cc-6bb2-4446-8e03-ad8b801c37b6.jpg)
